Library/Homebrew/cmd/update-bash.sh

```diff
@@ -2,22 +2,41 @@ brew() {
   "$HOMEBREW_BREW_FILE" "$@"
 }
 
+git() {
+  [[ -z "$HOMEBREW_GIT" ]] && HOMEBREW_GIT="$(which_git 2>/dev/null)"
```

Comment:
I wonder that we should add `set -e` in `bin/brew`. It's important especially in subshell.